VEHICLE/SetVehicleLightsMode.md

```diff
@@ -10,6 +10,8 @@ void _SET_VEHICLE_LIGHTS_MODE(Vehicle vehicle, int p1);
 ```
 
 ```
+Could be SET_VEHICLE_HEADLIGHT_SHADOWS?
```

Comment:
Any reason not to just rename this?

Reply:
not entirely sure about the name, was just a guess since its location kinda lined up, functionality wise i didn't test it

Reply:
Would be better off just not changing the file then imo.